storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -137,6 +137,13 @@
     @IsImplementationOfClass(implementsClass = IBlacklistStrategy.class)
     public static final String BLACKLIST_SCHEDULER_STRATEGY = "blacklist.scheduler.strategy";
 
+    /**
+     * Whether the blacklistscheduler will put a supervisor to blacklist when there is a bad slot/worker
```

Comment:
We could clarify this a little bit here. It's not really put into the blacklist directly because of a bad slot. 

Also in the javadoc, we can refer to blacklist scheduler using {@link org.apache.storm.scheduler.blacklist.BlacklistScheduler}.